services/ingest/app/routers/grafana.py

```
@@ -32,6 +34,7 @@ async def grafana_proxy(
     from_: str = Query(default="now-6h", alias="from"),
     to: str = Query(default="now"),
     theme: str = Query(default="light"),
+    _user: UserContext = Depends(get_current_user),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)